deepchem/models/torch_models/layers.py

```diff
@@ -6150,4 +6150,186 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         T_out = self.sigmoid_fn(self.T(x))
         output = H_out * T_out + x * (1 - T_out)
 
-        return output
```

Comment:
Some other changes not related to this PR are showing up, can you try fixing it? It might be due to some merging error.

Reply:
fixed, please re-review it